java-logging-appinsights/README.md

```diff
@@ -97,21 +97,18 @@ For custom telemetry metrics implement `AbstractAppInsights` already provided wi
 - CloudRoleNameInitializer
 
 `WebSyntheticRequestTelemetryInitializer` provides functionality to separate out requests via so called Synthetic Source tag.
-Unfortunately, custom headers are package private.
-In order to use this feature add following headers at will:
-
-- When `SyntheticTest-Source` is present:
-  - `SyntheticTest-Source` as Source name
-  - `SyntheticTest-UserId`
-  - `SyntheticTest-SessionId`
-  - `SyntheticTest-OperationId`
-  - `SyntheticTest-TestName`
-  - `SyntheticTest-RunId`
-  - `SyntheticTest-Location`
-- Otherwise:
-  - `Application Insights Availability Monitoring` as Source name
-  - `SyntheticTest-RunId`
-  - `SyntheticTest-Location`
+All headers are available in helper class [SyntheticHeaders](java-logging-appinsights/src/main/java/uk/gov/hmcts/reform/logging/appinsights/SyntheticHeaders).
```

Comment:
Would be nice to link to javadoc instead of having to repeat values inside this readme.

Reply:
One day